### PR TITLE
Replaced forum link

### PR DESCRIPTION
**Pull request by donielf**
With the forum having been deprecated I replaced the link with Meta.

### CODE REVIEW

**Comment by superplane39** [comment]
As long as we're updating the README, it probably needs a whole rewrite. The project long stopped being *"...a group of (mostly unsatisfied) users of Stack Overflow and the Stack Exchange Network to develop an alternative to these platforms..."*; it's far beyond that stage and has evolved into a separate project with different goals and aspirations.

Besides that, maybe changing the link text there would be a good idea? Maybe something like "check out our meta site" instead of forum?

But the README definitely needs more improvement than just a link change (although good catch that that's still in there).

File: README.md [hunk 1]
```diff
@@ -4,6 +4,6 @@ A work-in-progress, healthy alternative to the Stack Exchange Network.
 
 This project was created to coordinate and keep track of current efforts by a group of (mostly unsatisfied) users of Stack Overflow and the Stack Exchange Network to develop an alternative to these platforms.
 
-All relevant information can be found in our [wiki](https://github.com/codidact/docs/wiki). Please make sure to [check out the forum](https://forum.codidact.org) as well. Join our [Discord server](https://discord.gg/WZ7aTst) for live chat about the project.
+All relevant information can be found in our [wiki](https://github.com/codidact/docs/wiki). Please make sure to [check out the forum](https://meta.codidact.com) as well. Join our [Discord server](https://discord.gg/WZ7aTst) for live chat about the project.
```
Comment (cellio) [inline README.md:7]: Let's also change the link text to "check out our discussion area" or something similar.

File: README.md [hunk 2]
```diff
@@ -4,6 +4,6 @@ A work-in-progress, healthy alternative to the Stack Exchange Network.
 
 This project was created to coordinate and keep track of current efforts by a group of (mostly unsatisfied) users of Stack Overflow and the Stack Exchange Network to develop an alternative to these platforms.
```
Comment (cellio) [inline README.md:5]: While we're in there anyway, let's replace this with: This project was created to coordinate current efforts to develop a new community platform  (or something like that).

**Comment by sarahmccuan** [comment]
Closing as the entire landing page is being rewritten.

**Pull request closed without merging**